quick-start-eks.md

```diff
@@ -0,0 +1,117 @@
+# Quick start
+
+In this guide we will see how to start the [Initium Platform](https://github.com/nearform/initium-platform) to an EKS cluster and deploy an application to it from a GitHub action using the [Initium CLI](https://github.com/nearform/initium-cli).
+
+## Prerequisites
+
+- A working EKS cluster
+  - you can install the `eksctl` binary and create a cluster with it using the tools provided by `asdf` in `initium-platform` as explained below
+  - you'll need to give the node role permissions to manage EBS volumes
+- The EBS CSI driver installed on the cluster
+  - you can follow [these instructions](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) to get it installed
+- The cluster's control plane should be publicly exposed so the CLI can reach it
+  - remember to check the cluster's security groups
+
+## The Platform
+
+1. Clone the platform repository
+
+```bash
+git clone https://github.com/nearform/initium-platform.git
+```
+
+2. Install the required tooling
+
+```bash
+cd initium-platform
+make asdf_install
+```
+
+3. Create the EKS cluster (if you don't have one yet)
+
+```bash
+AWS_DEFAULT_REGION=<your default region> ~/.asdf/installs/eksctl/<eksctl version installed>/bin/eksctl create cluster
+```
+
```

Comment:
Does it make sense to add a page around creating a cluster here? Like the following one:
https://docs.aws.amazon.com/eks/latest/userguide/create-cluster.html